addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/CommandInjectionScanRuleUnitTest.java

```diff
@@ -73,6 +75,26 @@ protected CommandInjectionScanRule createScanner() {
         return scanner;
     }
 
+    @Test
+    void shouldReturnExpectedMappings() {
```

Comment:
```suggestion
    void shouldReturnExpectedAlertTags() {
```

Reply:
It also tests the CWE and WASC ids which is why I used that name ;)

Reply:
Hmmm are we going to deprecate the other details/methods or are they too ingrained?

Either way, that's fine leave it as-is.

Reply:
I def dont want to depreciate the CWE and WASC Id methods, at least not at this stage. I'm also not planning on adding tags for them yet either..